server/clip_server/executors/mclip_hg.py

```diff
@@ -0,0 +1,276 @@
+import os
+import warnings
+from multiprocessing.pool import ThreadPool
+from typing import Dict, Optional
+import numpy as np
+import torch
+from transformers import CLIPFeatureExtractor, CLIPModel, PreTrainedModel, PretrainedConfig, AutoTokenizer, AutoModel
+from clip_server.executors.helper import (
+    split_img_txt_da,
+    set_rank,
+)
+from jina import Executor, requests, DocumentArray
+
+
+class MCLIPConfig(PretrainedConfig):
+    model_type = "M-CLIP"
+
+    def __init__(self, modelBase='xlm-roberta-large', transformerDimSize=1024, imageDimSize=768, **kwargs):
```

Comment:
Don't use camel case for the variables/parameters, e.g., `transformerDimSize`. This name convention does not match the style in the current code base.